index.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
+
+// eslint-disable-next-line no-unused-vars
+
+// Hidde and display sections
+import {
+  addBooks,
+  contact,
+  addBoo,
+  contactHandler,
+} from './modules/hideDisplay.js';
+
+// class declaration
+
+import { BookCollection } from './modules/bookCollection.js';
+
+import { DateTime } from './luxon/build/es6/luxon.js';
```

Comment:
- [ ] kindly make sure to install `luxon` using `npm` instead and import the file in the modules folder